docs/DeveloperGuide.md

```diff
@@ -2,37 +2,159 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+The CS2113 Team (Professor Akshay <3) for the guidance and teaching us concepts of Software Engineering and OOP
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Main overview of the system:
 
+* Main Components
+  * Ui: Deals with user input/output
+  * Storage: Deals with data management on hard disk
+  * Parser: Deals with parsing user input
+  * Command: Deals with handling command input logic
+
+![broadOverview.png](UML%2FImage%2FbroadOverview.png)
+
+Fig 1
```

Comment:
Fig 1 should likely not be here similarly enough fig 2 down below recommend to rename.